docs/monitors/mysql_monitor.md

```diff
@@ -74,6 +74,10 @@ You can also use this data in [Dashboards](/help/dashboards) and [Alerts](/help/
 |||# ``database_hostport``||| Hostname (or IP address) and port number of the MySQL server, e.g. ``dbserver:3306``, or \
                               simply ``3306`` when connecting to the local machine. You should specify one of \
                               ``database_socket`` or ``database_hostport``, but not both.
+|||# ``use_ssl``          ||| Whether or not to use SSL when connecting to the MySQL server.
```

Comment:
What happens if this value is not provided. Does it use default system ca bundle? Or certifi one? Or something else?

Reply:
With no `use_ssl` it defaults to False and nothing is used. With no ca bundle defined it doesn't use one at all, it doesn't validate the server. Thats how mysql works for whatever reason but yeah it would be a good idea to clarify that here just in case people aren't aware.